forum/users/models.py

```diff
@@ -1,3 +1,26 @@
+from django.core.validators import EmailValidator
 from django.db import models
+import uuid
 
 # Create your models here.
+class Role(models.Model):
+    role_id = models.UUIDField(primary_key=True, default=uuid.uuid4)
+    role_name = models.CharField(max_length=50, null=False, unique=True)
+
+class User(models.Model):
+    user_id = models.UUIDField(primary_key=True, default=uuid.uuid4)
+    user_name = models.CharField(max_length=100, null=False, unique=True)
+    first_name = models.CharField(max_length=100, null=False)
+    last_name = models.CharField(max_length=100, null=False)
+    email = models.EmailField(unique=True, null=False, max_length=50, validators=[EmailValidator()])
```

Comment:
Using `EmailField` with `EmailValidator()` is redundant. The `EmailField` already includes basic email validation, so adding `EmailValidator()` is unnecessary.
Also, consider increasing the `max_length` for the email field. Standard practice is to use a maximum of 254 characters for email fields, as defined by `RFC 5321`.
https://datatracker.ietf.org/doc/html/rfc5321

Reply:
Removed validator from email field and set max_length=255.